src/controllers/index.mjs

```diff
@@ -98,7 +97,6 @@ initialize({
 
     { path: `${root}/labels`, module: labels },
     { path: `${root}/labels/{id}`, module: labelsId },
-    { path: `${root}/labels/{id}/albums`, module: labelAlbums },
```

Comment:
Don't know what the difference is between this endpoint and the `releases` endpoint, and also the code within it didn't do any filtering by label :shrug: